cloudformation/media-atoms-dynamo-prod.json

```diff
@@ -118,7 +123,47 @@
                     }
                 ]
             }
+        },
+        "UploadTrackingMediaAtomMakerProd": {
+          "Type": "AWS::DynamoDB::Table",
+          "Properties": {
+            "TableName": { "Ref": "UploadTrackingAtomMakerProdTable" },
+            "AttributeDefinitions": [
+              {
+                "AttributeName": "atomId",
+                "AttributeType": "S"
+              },
+              {
+                "AttributeName": "id",
+                "AttributeType": "S"
+              }
+            ],
+            "KeySchema": [
+              {
+                "AttributeName": "atomId",
+                "KeyType": "HASH"
+              },
+              {
+                "AttributeName": "id",
+                "KeyType": "RANGE"
+              }
+            ],
+            "ProvisionedThroughput": {
+              "ReadCapacityUnits": "5",
+              "WriteCapacityUnits": "5"
+            },
```

Comment:
I don't know so I copied what the other tables are using. I will be throttling the updates to the table to happen only when a part has finished uploading or every X seconds to update the progress of an upload.